CHANGELOG.md

```diff
@@ -1,3 +1,15 @@
+# Release 1.3.4
+
+# What's New
+
+* Moved `ziti ops verify-network` to `ziti ops verify network`
+* Moved `ziti ops verify traffic` to `ziti ops verify traffic`
+* Added `ziti ops verify ext-jwt-signer oidc` to help users with configuring OIDC external auth 
```

Comment:
This breaks backward compatibility. We usually care about that for other commands. These though? I don't feel strongly as they are debugging tools.